Documentation/guides/port.rst

```diff
@@ -0,0 +1,62 @@
+.. include:: /substitutions.rst
+===========================================
+How to port
+===========================================
+
+This guide explains "How to port NuttX".  At this guide, the scope of porting is to add new SoC/Board directory, the goal of porting is to comfirm to boot NuttShell(NSH).
+
+The poring NuttX mainly consists of processor arch directory, SoC directory and Board directory.
```

Comment:
"Porting a new SoC and board to NuttX requires modification on arch, SoC and board directories."